.gitignore

```diff
@@ -16,7 +16,8 @@ env/
 venv/
 ENV/
 VENV/
-
+hnn_core/mod/*
+test.*
```

Comment:
I'm not sure about this. Why are you adding this type of files?

Reply:
Apologies for the oversight; it was committed accidentally. Let me remove it